elasticdl/doc/tutorials/101.md

```diff
@@ -0,0 +1,35 @@
+# 101 Tutorial
```

Comment:
I feel like it's better to be called "Getting Started with ElasticDL" similar to https://www.tensorflow.org/tutorials. "101" seems a bit informal and people from other countries may not understand what this means.

Reply:
Done